src/documentation/netex_idfm_to_ntfs.md

```diff
@@ -0,0 +1,292 @@
+and transformed into a [NTFS feed](https://github.com/CanalTP/ntfs-specification/blob/master/ntfs_fr.md).
+
+For the sake of simplicity, the NTM properties that are not specified in the source
+data are not described below.
+
+In order to guarantee that the NTFS objects identifiers are unique and stable, each
+object id is prefixed with a unique prefix (specified for each datasource), following
+the general pattern `<prefix>:<id>`.
+
+## Input data description
+This specifications assumes that all the requeried data (time tables for all the lines, stop points and stop areas, transfers, etc.) are provided in one ZIP archive (aka "FICHIERS OFFRE") described in the specification document "NT60-A150701-v1.11-BO-STIF_-_Specification_Technique_d_Interface_NeTEx_pour_la_publication_20190624.docx".
+
+The ZIP archive contains contains: 
+- a **arrets.xml** file
+- a **lignes.xml** file, containing the description of lines, networks and companies
+- a **correspondances.xml** file
+- a folder for each operator (or company) containing:
+  + a **calendriers.xml** file, containing the calendars (or validity patterns) used by trips in files **offre_**
+  + a **commun.xml** file optionnal, containing the comments referenced by the operator objects (if needed)
+  + a file starting with **offre_** describing the time tables of a specific line
+
+In this document, versions of objects are not handled. The first encountered object description is considered when creating an object.
+
+Each XML files contains a `PublicationDelivery` node, containing a `dataObjects` node. Descriptions below is considering nodes inside this `dataObjects` node.
+
+## Reading of the "arrets.xml" file
+### stops.txt
+`stops` are provided in the **arrets.xml** file in the node **CompositeFrame/frames/GeneralFrame/** (only one **CompositeFrame** is expected). 
+In this netex feed, a `Quay` in included in a "ZDL" `StopPlace`, this "ZDL" `StopPlace` could be included in a "LDA" `StopPlace`.
+This connector assumes that all the "ZDL" `StopPlace` are included in one (and only one) "LDA" `StopPlace`.
+
+#### For stop_areas
+`stop_area` objects are `StopPlace` nodes of the **arrets.xml** file created in this order:
+1. `StopPlace` with a `placeTypes/TypeOfPlace/@ref` attribute with a "LDA" value are used
+2. Other `StopPlace` with no `ParentSiteRef` node (corresponding to "ZDL" stops not included in a LDA)
+
+NTFS field | Netex-IDFM element | Mapping rule/Comment
+--- | --- | ---
+stop_id | *StopPlace/@id* | This field is prefixed. 
+stop_name | *StopPlace/Name* | 
+location_type | | Fixed value `1` (stop_area)
+stop_lat | Quay/Centroid/Location | see (1) below
+stop_lon | Quay/Centroid/Location | see (1) below
+
+(1) Definition of stop_lat et stop_lon:
+As this Netex feed does not provide coordinates for the stop_areas, the stop_lat et stop_lon fields will be set with the coordinate of the centroid of all included stop_points.
+
+(2) Complementary object_properties
```

Comment:
Notes 2 and 3 are not referenced in the table above. Maybe separate as subsections?